searx/engines/libretranslate.py

```diff
@@ -31,13 +31,19 @@ def request(_query, params):
 
     params['method'] = 'POST'
     params['headers'] = {'Content-Type': 'application/json'}
+    params["raise_for_httperror"] = False
 
     return params
 
 
 def response(resp):
-    json_resp = resp.json()
 
-    result = {'infobox': json_resp['translatedText'], 'content': ''}
+    results = []
+    if resp.ok:
+        json_resp = resp.json()
+        text = json_resp.get('translatedText')
+        if text:
+            # results.append({'infobox': text, 'content': ''})
+            results.append({'answer': text})
```

Comment:
Personally I prefer to have the translation in the *answers box* .. but I'm not sure if this is a really good idea / please try it out and rate it

Reply:
I've applied this for the engines that don't use HTML now - the ones using HTML display more detailed information that are like a dictionary, so they probably fit well in the infobox :+1: